enhancements/insights/insights-operator-runtime.md

```diff
@@ -0,0 +1,532 @@
+
+Exposing these data to the user with a clean user interface (eg within console.redhat.com) is out of scope of this proposal.
+
+### Drawbacks
+
+If implemented, Red Hat will document the addition of this new collection. We  introduce the risk that some number of customers who would have or are sending telemetry data via the Insights operator will choose to turn it off because they are uncomfortable sharing this additional data. 
+
+The insights-operator-runtime needs to be continually updated to detect new workloads  that Red Hat wants information about. 
+This can be done iteratively and will require an updated reference to the insights-operator-runtime image in the Insights Operator to take into account the additional detections.
+
+As the insights-operator-runtime is sanitizing the data extracted from the containers, it is not able to retrieve "raw" data (such as processes' name and command line) which limits its ability to detect emerging trends. The insights-operator-runtime must provide fingerprints executables for workloads that Red Hat wants to monitor.
+
+## Design Details
+
+### Open Questions [optional]
+
+#### insights-operator-runtime Image Availability
+
+The Insights Operator must be able to deploy the insights-operator-runtime from an image that is available by default in the cluster image registry.
+
+
+## Test Plan
+
+### insights-operator-runtime Test Plan
+
+The insights-operator-runtime can be tested in isolation to verify its capability to scan containers and collect the expected runtime info.
+
+Testing the insights-operator-runtime can be done outside of the Insights Operator test suite.
+It requires integration tests with applications based on the detectable runtime stacks. A end-to-end (e2e) test suite can be developed to verify the capabilities of the insights-operator-runtime.
+
+### Insights Operator Test Plan
+
+The integration tests for the Insights Operator will verify that it is able to properly deploy the insights-operator-runtime as a `DaemonSet` and have some smoke tests to validate that the additional runtime info can effectively be collected.
+
+## Graduation Criteria
+
+TBD
+
+### Dev Preview -> Tech Preview
+
+This enhancement would be behind the featuregate capabilities of OpenShift
+
+* Add a non-public flag to enable it on a cluster to collect runtime information
+* End User documentation is not required
+* Sufficient test coverage
+* Gather feedback from users
+* Generate reports to validate the consumption of the additional runtime information
+
+### Tech Preview -> GA
+
+*  More testing
+*  Sufficient time for feedback
+*  Available by default
+*  Conduct load testing
+* User facing documentation created in [openshift-docs](https://github.com/openshift/openshift-docs/blob/main/support/remote_health_monitoring/using-insights-operator.adoc) if needed
+
+### Removing a deprecated feature
+
+Not relevant for this proposal
```

Comment:
It might still be a good idea to list all required actions if this feature is removed.